packages/flutter_tools/test/general.shard/flutter_tester_device_test.dart

```diff
@@ -12,6 +12,8 @@ import 'package:flutter_tools/src/base/logger.dart';
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/build_info.dart';
 import 'package:flutter_tools/src/device.dart';
+import 'package:flutter_tools/src/native_assets.dart';
```

Comment:
This would break the roll into G3.

Instead add a new test in `test/general.shard/isolated/` that dir is excluded in the import.

For more info see: https://github.com/flutter/flutter/pull/143055